### PR TITLE
New post: why workflows matter

### PR DESCRIPTION
**Pull request by crichID**
@githubteacher per the conversation in #9, starting a new post to discuss the importance of crafting awesome (and simple) workflows.

### CODE REVIEW

File: _posts/2017-02-13-workflows-matter.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout:     post
+title:      "Why Your Workflow Matters"
```
Comment (githubteacher) [inline _posts/2017-02-13-workflows-matter.md:3]: @crichID I think this is a fabulous title!

File: _posts/2017-02-13-workflows-matter.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+subtitle:   "and your developers know it."
+date:       2017-02-13 12:00:00
+author:     "GitHub Teacher"
+header-img: "img/post-bg-01.jpg"
+---
+
+<p>Complicated workflows make life miserable, simple workflows make developers happy!</p>
+
+<p><a href="http://broken.broken.broken/">A simple Git workflow</a></p>
```
Comment (githubteacher) [inline _posts/2017-02-13-workflows-matter.md]: @crichID I think this link is broken. Do you mind looking into it?

**Review by githubteacher — CHANGES_REQUESTED** [review]
Almost there - just a link to review, and maybe 2 given Travis's output. 🤔

**Comment by crichID** [comment]
@githubteacher I fixed the URL in the post!

**Review by githubteacher — APPROVED** [review]

**Pull request merged**